src/fuzzing/src/test/scala/Fuzzing.scala

```diff
@@ -79,33 +80,42 @@ class Fuzzing extends TestBase {
             createDataSet
           }
         pipelineModel.transform(dfTransform)
+        println(s"Successful round trip with $estimatorName")
         ()
       })
+
     })
   }
 
   test("Verify all transformers can be turned into pipelines, saved and loaded") {
-    transformers.foreach(tr => {
+    val exemptions: Set[String] = Set(
+      "com.microsoft.ml.spark.EnsembleByKey",
+      "com.microsoft.ml.spark.MultiColumnAdapter",
+      "com.microsoft.ml.spark.CNTKModel",
+      "com.microsoft.ml.spark.IndexToValue",
+      "com.microsoft.ml.spark.ComputeModelStatistics"
+    )
+
+    val applicableTransformers = transformers.filter(t => !exemptions(t.getClass.getName))
+
+    applicableTransformers.foreach(tr => {
       val transformerName = tr.getClass.getName
-      println()
-      println(s"Running transformer: ${tr.toString} with name: ${transformerName}")
       val (dataset, pipelineStage) =
         if (transformerFuzzers.contains(transformerName)) {
-          println("Generating dataset from transformer fuzzer")
           val transformerFuzzer = transformerFuzzers(transformerName)
           val fitDataset = transformerFuzzer.createDataset
           val trUpdated = transformerFuzzer.setParams(fitDataset, tr.copy(ParamMap()))
           (fitDataset, trUpdated.asInstanceOf[PipelineStage])
         } else {
-          println("Generating random dataset")
           (createDataSet, tr.copy(ParamMap()).asInstanceOf[PipelineStage])
         }
       tryRun(() => {
         val pipeline = new Pipeline().setStages(Array(pipelineStage))
         val pipelineModel = pipeline.fit(dataset)
         trySave(pipelineModel)
+        println(s"Sucessful round trip with $transformerName")
         ()
-      })
+      }, () => println(s"round trip failed with $transformerName"))
```

Comment:
You can probably programatically generate a test case so failures show up atomically and get registered by JUnit... Please considering doing this when refactoring the fuzzing system.